docs/products/access/api-reference/config-api/identity-provider-api.md

```diff
@@ -93,13 +94,104 @@ Pragma: no-cache
+| tulip_api_access_scope       | TULIP        | depends  | Space-separated scopes for the required Tulip segments e.g. `iwelcome:segment:example`. <br/> Required when `APP_TO_WEB` or `UDH_API` integration is enabled.                                                           |
+| authorization_url            | OAUTH        | yes      | Oauth authorization endpoint.                                                                                                                                                                                           |
+| token_url                    | OAUTH        | yes      | Oauth token endpoint.                                                                                                                                                                                                   |
+| profile_url                  | OAUTH        | yes      | OpenID Connect UserInfo endpoint.                                                                                                                                                                                       |
+| user_info_enabled            | OAUTH        | no       | Specify whether CIM's Person API is enabled for this Identity Provider. <br/> Default value: `false`.                                                                                                                   |
+| user_info_endpoint           | OAUTH        | depends  | Identity source URL. The URL of API that provides user's identity. Use `{userId}` placeholder for userId path param. e.g. https://host/api/persons/{userId}/profile  <br/> Required when `user_info_enabled` is `true`. |
+| user_info_username           | OAUTH        | depends  | Identity source username. <br/> Required when `user_info_enabled` is `true`.                                                                                                                                            |
+| user_info_password           | OAUTH        | depends  | Identity source password. <br/> Required when `user_info_enabled` is `true`.                                                                                                                                            |
+
+Example `TULIP` type request:
+
+```http
+POST /api/v1/configuration/idps 
+Host: onegini.example.com
+Content-Type: application/json
+{
+   "id":"tulip-insurcar",
+   "name":"Tulip Insurcar Idp",
+   "type":"TULIP",
+   "client_id":"oauth2CustomerApp",
+   "issuer_uri":"https://insurgroup-edge.iwelcome.com/insurcar/login/oauth2/v1",
+   "end_session_enabled":true,
+   "scopes":"openid",
+   "integrations":[
+      "APP_TO_WEB",
+      "UDH_API"
+   ],
+   "tulip_api_client_id":"accessIntegration",
+   "tulip_api_base_url":"https://insurgroup-edge.iwelcome.com/insurcar",
+   "tulip_api_access_scope":"iwelcome:segment:insurgroup"
```

Comment:
removed some "real looking urls" and also added spaces for readability in the json

```suggestion
   "id": "tulip-brand",
   "name": "Tulip Brand",
   "type": "TULIP",
   "client_id": "oauth2CustomerApp",
   "issuer_uri": "https://tulip.onewelcome.com/brand/login/oauth2/v1",
   "end_session_enabled": true,
   "scopes": "openid",
   "integrations": [
      "APP_TO_WEB",
      "UDH_API"
   ],
   "tulip_api_client_id": "accessIntegration",
   "tulip_api_base_url": "https://tulip-api.onewelcome.com/brand",
   "tulip_api_access_scope": "iwelcome:segment:brand"
```

Reply:
Good point! Thanks!